_data/simple-icons.json

```diff
@@ -3648,6 +3648,11 @@
             "hex": "4285F4",
             "source": "https://fonts.gstatic.com/s/i/productlogos/calendar_2020q4/v8/192px.svg"
         },
+        {
+            "title": "Google Campaign Manager",
+            "hex": "5BB974",
+            "source": "https://www.gstatic.com/analytics-suite/header/suite/v2/ic_campaign_manager.svg"
```

Comment:
I would suggest we need to find a better source than a static SVG. Where is this SVG visible?